docs/src/main/tut/docs/play/README.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: docs
+title: Play Framework
+permalink: /docs/play/
+---
+
+# Play Framework integration
+
+It is easy to use a freestyle program as a result of a Play Framework Action with the _freestyle-play_ module. This module provides an implicit conversion `FreeS[F, A] => Future[A]` which allows a user to define a Free program as the result of any Play Action that expects a Future as a response.
+
+In order to enable this integration you may depend on _freestyle-play_
+
+```scala
+libraryDependencies += "com.47deg" %% "freestyle-play" % "0.1.0"
```

Comment:
It seems the other http integrations are `freestyle-http-akka`, `freestyle-http-finch` and `freestyle-http-http4s`.

Should we rename `freestyle-play` to `freestyle-http-play` to be consistent? (or the other way around?)